html_app/scb_ex/AddMultipleDialog.js

```diff
@@ -160,20 +206,37 @@ scb_ex2.setup=function(state){
         dialog.css(k, v);
     });
 
-    $('.scb_ex_dialog').draggable({handle: '.scb_ex_inner_dialog_title'})
+    $('.scb_ex_dialog').draggable({handle: '.scb_ex_inner_dialog_title'});
 
 }
-scb_ex3.setup=function(state){
+scb_ex3.setup=function(state) {
     var workarea = state.workarea;
-    var assignment=state.assignment;
-    var experiment=state.experiment;
-    var  template = state.template;
-    var dialog=$("<div class='scb_ex_dialog'></div>");
+    var assignment = state.assignment;
+    var experiment = state.experiment;
+    var template = state.template;
+    var dialog = $("<div class='scb_ex_dialog'></div>");
     dialog.html(scb_ex.dialog({
-       assignment: assignment,
+        assignment: assignment,
         experiment: experiment,
         template: template
     }));
+}
+/*Ivan's version*/
```

Comment:
Really this is the only version...either more explanation as to why that matters, or remove the comment